### ISSUE TITLE
Add option to limit download speed

### ISSUE THREAD

**Issue by Ragzouken** [itchio/itch#1123]
So you can make sure it doesn't eat the whole house's capacity. Always wished steam had this.

**Comment by fasterthanlime** [comment]
> Always wished steam had this.

Actually I think it does! As for itch, it's planned as well - I think another issue mentioned this already.

**Comment by sigmap** [comment]
I'd like to see this in itch.

**Comment by fasterthanlime** [comment]
A quick update: itch v25 has code for throttling the overall bandwidth usage, but it's not accessible anywhere in the UI yet.

**Comment by omgitsraven** [comment]
Is it accessible in a config file or something?

**Comment by fasterthanlime** [comment]
@omgitsraven nope, no setting is accessible anywhere right now, not even via config files.

**Comment by insomniacUNDERSCORElemon** [comment]
@fasterthanlime any update on this? Also what about better bundle support than the website (issue 2401)

Also forgive me for off-topic question: for outside-of-client itch games, is it as easy as moving files to add them to the client?
